Tools/cases_generator/generate_cases.py

```diff
@@ -15,14 +15,14 @@
 import parser
 from parser import StackEffect
 
-DEFAULT_INPUT = os.path.relpath(
-    os.path.join(os.path.dirname(__file__), "../../Python/bytecodes.c")
-)
-DEFAULT_OUTPUT = os.path.relpath(
-    os.path.join(os.path.dirname(__file__), "../../Python/generated_cases.c.h")
-)
+HERE = os.path.dirname(__file__)
+ROOT = os.path.join(HERE, "../..")
+THIS = os.path.relpath(__file__, ROOT)
+
+DEFAULT_INPUT = os.path.relpath(os.path.join(ROOT, "Python/bytecodes.c"))
+DEFAULT_OUTPUT = os.path.relpath(os.path.join(ROOT, "Python/generated_cases.c.h"))
```

Comment:
```suggestion
DEFAULT_INPUT = os.path.relpath(os.path.join(ROOT, "Python", "bytecodes.c"))
DEFAULT_OUTPUT = os.path.relpath(os.path.join(ROOT, "Python", "generated_cases.c.h"))
```

Reply:
This also feels a bit pedantic and less readable...